src/_layouts/page.html

```diff
@@ -1,6 +0,0 @@
----
```

Comment:
There are still a few pages referring to `layout: page`, you might want to strip those out now.

Reply:
I can't find any references to `layout: page` actually.

Reply:
I thought that I had seen some in this PR (although I thought that I had taken them all out). If there are none left, then good!